code/modules/admin/secrets.dm

```diff
@@ -639,6 +640,15 @@ GLOBAL_DATUM_INIT(admin_secrets, /datum/admin_secrets, new)
 					M.req_access = list()
 					M.req_one_access = list(ACCESS_BRIG,ACCESS_ENGINE)
 			message_admins("[key_name_admin(usr)] made all maint doors engineering and brig access-only.")
+		if("maint_access_normal") // allows an admin to revert the effects of maint_access_brig or maint_access_engiebrig
+			if(!check_rights(R_DEBUG))
+				return
+			for(var/obj/machinery/door/airlock/maintenance/M in GLOB.machines)
```

Comment:
You mean this line?

I was assuming that modifying the permission of a maint door (var/obj/machinery/door/airlock/maintenance) does not modify the type of the item itself, so this _should_ work to fix it.

I am incorrect in that?

Reply:
check_access() resets airlocks accesses to their default anyway

Reply:
Modifying the permission of a maint door would modify the permission of the maint door for that object, which is the thing you are checking

Reply:
Nevermind check_access doesn't reset access meaning you are checking M.req_access for doors that don't use that access after you run the brig access only thing